tun/Cargo.toml

```diff
@@ -10,13 +10,18 @@ nix = { version = "0.26", features = ["ioctl"] }
 socket2 = "0.4"
 tokio = { version = "1.28", features = [] }
 byteorder = "1.4"
+env_logger = "0.10.0"
```

Comment:
I think `env_logger` should go in `burrow` and not `tun` because `tun` just needs the logging frontend

I also believe you need to call `env_logger::init();` in burrow's `main` function